lib/src/select/Listbox.tsx

```diff
@@ -23,25 +21,24 @@ const Listbox = ({
 }: ListboxProps): JSX.Element => {
   const translatedLabels = useTranslatedLabels();
   const listboxRef = useRef(null);
-  const listboxId = `select-${useId()}`;
-  
+
   let globalIndex = optional && !multiple ? 0 : -1;
   const mapOptionFunc = (option, mapIndex) => {
-    const groupId = `${listboxId}-group-${mapIndex}`;
+    const groupId = `${id}-group-${mapIndex}`;
     if (option.options) {
       return (
         option.options.length > 0 && (
           <li key={groupId}>
-            <GroupList role="listbox" aria-labelledby={groupId}>
+            <ul role="listbox" aria-labelledby={groupId} style={{ padding: 0 }}>
```

Comment:
The main benefits of styled components don't apply to the use cases I used inline components. There is a small performance cost for using styled components due to the creation of dynamic styles and their processing by the library. Also, the code will not scale over time and having it inline helps with at-a-glance compression of the component code (for example, we can understand at first glance what the input is used for).

The only concession is the consistency with the rest of the code, but I don't see a bad practice to try to avoid using so many styled components, whenever possible.

Reply:
In this scenario, it was more of a consistency concern than performance. I understand your point though.
Nevertheless I consider that we should have a quick meeting to decide what the criteria should be when we have to decide between styled components and inline styles.